app.yaml.template

```diff
@@ -0,0 +1,33 @@
+# Google App Engine Config
+
+runtime: python37
+
+# The --http flag exposes the app with uWSGI only and will act as the router/proxy/load balancer
+# If we want an nginx webserver upstream for more features, we would have to use a --socket flag so that nginx can communicate with uWSGI
+# More info here: https://uwsgi-docs.readthedocs.io/en/latest/tutorials/Django_and_nginx.html
+entrypoint: uwsgi --http :8080 --module tcf_core.wsgi --master --processes 1 --threads 2
+
+handlers:
+# This configures Google App Engine to serve the files in the app's static
+# directory.
+- url: /static
+  static_dir: tcf_website/static/
```

Comment:
do you know if we have to do `collectstatic` and then set `static_dir` to `STATIC_ROOT`? https://docs.djangoproject.com/en/3.0/howto/static-files/#deployment

Reply:
NVM I see that you changed `STATIC_ROOT`. do you think we need to run `collectstatic`?

Reply:
Yep you are right, I need to add `collectstatic`. Looking at the link I think there are a couple options we could use: 

- If we want to serve the static files from the same server, then we just run collectstatic before deploying and keep `STATIC_ROOT` as it is right now

- Or if we want a separate designated server for serving static files, we'll have to find a way to run `collectstatic` and push those files to that server. 

I personally think we don't need another server but what do you think?

Reply:
I think we're good with 1 server for now. If needed, we can put them in a bucket or something later.

Reply:
Okay sounds good. I will add that `collectstatic`